src/tabs/styled-components.js

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles/index.js';
+import type {SharedStylePropsT} from './types.js';
+import {ORIENTATION} from './constants.js';
+
+/**
+ * Main component container element
+ */
+export const Root = styled('div', (props: SharedStylePropsT) => {
+  const {$orientation} = props;
+  return {
+    display: 'flex',
+    flexDirection: $orientation === ORIENTATION.vertical ? 'row' : 'column',
+  };
+});
+
+export const Tab = styled('div', (props: SharedStylePropsT) => {
+  const {
+    $disabled,
+    $active,
+    $orientation,
+    $theme: {colors, sizing, typography},
+  } = props;
+  let style = {
+    ...typography.font300,
+    boxSizing: 'border-box',
+    color: $active ? colors.black : colors.mono800,
+    cursor: $disabled ? 'not-allowed' : 'pointer',
+    height: sizing.scale1200,
+    paddingTop: sizing.scale550,
+    paddingLeft: sizing.scale300,
+    paddingRight: sizing.scale300,
+    marginLeft: sizing.scale200,
+    marginRight: sizing.scale200,
+    borderBottom:
+      $orientation === ORIENTATION.horizontal && $active
+        ? `2px solid ${colors.primary400}`
+        : 'none',
```

Comment:
How about for non-active tabs we still set the border to `2px` but make the color `transparent`? I see that there is no height jump since we set a fixed height, but for consistency seems to be a good approach and for the case when the height of the tabs is overwritten to be not fixed.

Reply:
Removed the fixed height. 
`boxSizing: 'border-box'` accomplishes the same thing right?